go/libkb/id_table.go

```diff
@@ -769,6 +772,53 @@ func (s *PerUserKeyChainLink) ToPerUserKey() keybase1.PerUserKey {
 	}
 }
 
+//-------------------------------------
+
+// VerifyReverseSig checks a SibkeyChainLink's reverse signature using the ComputedKeyFamily provided.
+func (s *PerUserKeyChainLink) VerifyReverseSig(_ ComputedKeyFamily) (err error) {
```

Comment:
any way to reuse the `Sibkey` version of this?  there are very few differences so far as i can tell?  it seems like after you get the key, it's the same thing...  factor out into a helper?